app/src/main/java/nodomain/freeyourgadget/gadgetbridge/activities/FwAppInstallerActivity.java

```diff
@@ -128,6 +148,12 @@ private void validateInstallation() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        if (GBApplication.isDarkThemeEnabled()) {
```

Comment:
I still need to check if the dark theme is enabled and change the theme if it is. But the else statement is redundant